mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -610,6 +612,10 @@ func expandAlertConfigurationThresholdConfig(d *schema.ResourceData) *matlas.Thr
 		vL := value.([]interface{})
 
 		if len(vL) > 0 {
+			if vL[0] == nil {
+				return nil
+			}
+
 			v := vL[0].(map[string]interface{})
```

Comment:
I think I tried to do it but the `ok` was still true but I am not 100% sure now 🤦 . I will add an update in a few minutes to see if this would work. Thanks for flagging this

Reply:
Yes, We can use the `ok,v `pattern here. I opened https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1420 to address this comment. Thanks